document/avatar/avatar.css

```diff
@@ -0,0 +1,65 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
```

Comment:
If you have repeated this part in every file. I would suggest to move to base file of documentation CSS file. Rest is all good